### ISSUE TITLE
list venues that are open for hosting user group meetings

### ISSUE THREAD

**Issue by coderbyheart** [coderbyheart/ugrm#14]

**Comment by coderbyheart** [comment]
@ookami86 we could use one XML file with location entries.

Based on that I can render the list on the ugrm site.

**Comment by Ookami86** [comment]
Yeah, I thought of something like https://gist.github.com/Ookami86/a27074f9a802b4ce3396 can you think of anything else of interest to organisers?

**Comment by coderbyheart** [comment]
A photo and or logo.
Proper address (street, zipcode). I'd just use the location element from [user group.xsd](https://github.com/coderbyheart/ugrm-data/blob/master/xsd/usergroup.xsd#L285)

**Comment by coderbyheart** [comment]
@Ookami86 I've added schema for venues in coderbyheart/ugrm-data#115. Example added for [Zentrale](https://github.com/coderbyheart/ugrm-data/blob/f12b39ff7ebb74334152d21e38bf5e45909b6dd7/venue/zentrale.xml).

Once we have 5 venues, I'm implementing the frontend for it.

**Comment by Ookami86** [comment]
Cool, thanks
